src/main/webapp/app/course/course-scores/course-scores-csv-row-builder.ts

```diff
@@ -41,7 +41,11 @@ export class CourseScoresCsvRowBuilder {
      * @param value That should be placed in the row. Replaced by the empty string if undefined.
      */
     set<T>(key: string, value: T) {
-        this.csvRow[key] = value ?? '';
```

Comment:
Why is the ??-operator not possible here?

Reply:
I did not intend that change and it is a left-over from something else I was testing. Reverted in the latest commit.